crucible-llvm/src/Lang/Crucible/LLVM/MemModel/Value.hs

```diff
@@ -120,6 +121,23 @@ llvmValStorableType v =
     LLVMValZero tp -> tp
     LLVMValUndef tp -> tp
 
+freshLLVMVal :: IsSymInterface sym =>
+                sym -> StorageType -> IO (LLVMVal sym)
+freshLLVMVal sym tp =
+  case storageTypeF tp of
+    Bitvector bytes ->
+      case mkNatRepr (bytesToBits bytes) of
+        Some repr ->
+          case isPosNat repr of
+            Just LeqProof -> LLVMValInt <$> freshNat sym emptySymbol
```

Comment:
I think the block number should just be `0` in this case.  This corresponds to treating the freshly-created value as a "raw" bitvector, which I think is what we want.

Reply:
This means that anything we read from uninitialized memory will not be a dereferencable pointer, which seems fine.

Reply:
Good suggestion.